docker/ee.juicefs.Dockerfile

```diff
@@ -42,3 +42,6 @@ RUN jfs_mount_path=${JFS_MOUNT_PATH} && \
     chmod +x ${jfs_mount_path} && cp juicefs.py ${JUICEFS_CLI} && chmod +x ${JUICEFS_CLI}
 
 RUN /usr/bin/juicefs version
+
+ENV K8S_VERSION v1.14.8
+RUN curl -o /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/${TARGETARCH}/kubectl && chmod +x /usr/local/bin/kubectl
```

Comment:
why we need `kubectl` in mount image

Reply:
It's need in fuse image which is for Fluid. This PR merged the two in order to better maintain the image in the future.